src/js/contentControl.js

```diff
@@ -17,10 +10,10 @@ module.exports = function() {
     function hideData() {
         // hide the data sections
         // these get shown as needed in addresses.js (render)
-        r.addClass('hide');
-        nR.addClass('hide');
-        nF.addClass('hide');
-        dup.addClass('hide');
+        $('#rural').addClass('hide');
+        $('#notRural').addClass('hide');
+        $('#duplicate').addClass('hide');
+        $('#notFound').addClass('hide');
```

Comment:
Were the saved nodes getting replaced due to renders? So your references were to nodes out of the live DOM?